doc/src/modules/combinatorics/pc_groups.rst

```diff
@@ -184,16 +184,19 @@ To get a clear picture, start with an example of SymmetricGroup(4). For S(4) the
 generators in pcgs say [x0, x1, x2, x3] and the relative_order vector is [2, 3, 2, 2].
 Starting from bottom of this sequence the presentation is computed in order as below.
 
->>> x3^2	   	---|
->>> x2^2	      	   |---> using only [x3] from pcgs and pc_series[1]
->>> x2^{-1}x3x2		---|
->>> x1^3	   	---|	
->>> x1^{-1}x3x1		   |---> using [x3, x2] from pcgs and pc_series[2] 
->>> x1^{-1}x2x1		---|
->>> x0^2	   	---|
->>> x0^{-1}x3x0		   |---> using [x3, x2, x1] from pcgs and pc_series[3]
->>> x0^{-1}x2x0     	   |
->>> x0^{-1}x1x0		---|
+>>> x3^2               # ---|---> using only [x3] from pcgs and pc_series[4]
```

Comment:
It seems that `>>>` should not be included. Sphinx will try to run the code.

Reply:
Okay, If we'll write it directly then, how we can use comments because I tried it without `>>>` but it generates an error due to the comments(basically due to`|-->`), or may be we can write those comments directly without using special symbols.

Reply:
Maybe it is `|` that is causing the problems. `#` will probably not be needed if this is not run.

Reply:
It doesn't look good without `>>>` and keeping the above structure so, I tried removing those symbols and putting the comments up, have a look at the last commit, if it looks good.